### ISSUE TITLE
building 1.0.7 fails on OSX 10.10.3

### ISSUE THREAD

**Issue by mdemare** [eventmachine/eventmachine#590]
This is with version 6.3 of the command line tools.

```
Fetching: eventmachine-1.0.7.gem (100%)
Building native extensions.  This could take a while...
ERROR:  Error installing eventmachine:
	ERROR: Failed to build gem native extension.

    /Users/mdemare/.rbenv/versions/2.2.1/bin/ruby -r ./siteconf20150414-36225-1uemewc.rb extconf.rb
checking for rb_trap_immediate in ruby.h,rubysig.h... no
checking for rb_thread_blocking_region()... no
checking for ruby/thread.h... yes
checking for rb_thread_call_without_gvl() in ruby/thread.h... yes
checking for inotify_init() in sys/inotify.h... no
checking for __NR_inotify_init in sys/syscall.h... no
checking for writev() in sys/uio.h... yes
checking for rb_thread_fd_select()... yes
checking for rb_fdset_t in ruby/intern.h... yes
checking for rb_wait_for_single_fd()... yes
checking for rb_enable_interrupt()... no
checking for rb_time_new()... yes
checking for sys/event.h... yes
checking for sys/queue.h... yes
checking for clock_gettime()... no
checking for gethrtime()... no
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
compiling binder.cpp
In file included from binder.cpp:20:
In file included from ./project.h:29:
In file included from /Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/iostream:38:
In file included from /Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/ios:216:
In file included from /Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/__locale:15:
In file included from /Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/string:439:
In file included from /Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:628:
In file included from /Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/memory:604:
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/iterator:341:10: fatal error: '__debug' file not found
#include <__debug>
         ^
1 error generated.
make: *** [binder.o] Error 1

make failed, exit code 2

Gem files will remain installed in /Users/mdemare/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/eventmachine-1.0.7 for inspection.
Results logged to /Users/mdemare/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/extensions/x86_64-darwin-14/2.2.0-static/eventmachine-1.0.7/gem_make.out
```

**Comment by erichs** [comment]
As a hack workaround, you can do this:

https://github.com/JuliaLang/julia/issues/10822#issuecomment-93124394

**Comment by maksimf** [comment]
Same issue

**Comment by balinterdi** [comment]
What has worked for me is downgrading my Apple Command Line Tools to (XCode) 6.2. This is not a true solution but gets the job done (I hope for you, too).

**Comment by maksimf** [comment]
@balinterdi yep, that fixed, thanks

**Comment by bmishkin** [comment]
as noted in the last comment here

http://stackoverflow.com/questions/29529455/missing-c-header-debug-after-updating-osx-command-line-tools-6-3

this has been fixed in command line tools 6.3.1 (labeled as a second "6.3" in app store updates). 

I can verify that eventmachine version `1.0.7` (as well as `1.0.3`) now installs cleanly under OS X 10.10.3 after installing the command line tools update.

**Comment by sodabrew** [comment]
Excellent! Glad that Apple pushed out a fixed package before too long.
